pkg/backend/watch_darwin_arm64.go

```diff
@@ -21,6 +21,6 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/util/result"
 )
 
-func Watch(ctx context.Context, b Backend, stack Stack, op UpdateOperation, apply Applier, paths []string) result.Result {
+func Watch(ctx context.Context, b Backend, s Stack, op UpdateOperation, apply Applier, paths []string) result.Result {
```

Comment:
This change is to make my local linter happy because the line was 122 characters